src/shared/OutbreakProvider.tsx

```diff
@@ -1,22 +1,47 @@
+import {TEST_MODE} from 'env';
 import AsyncStorage from '@react-native-community/async-storage';
 import React, {useContext, useEffect, useMemo, useState} from 'react';
 import {Key} from 'services/StorageService';
 import PushNotification from 'bridge/PushNotification';
 import {useI18nRef, I18n} from 'locale';
+import PQueue from 'p-queue';
+
+// eslint-disable-next-line @shopify/strict-component-boundaries
+import {DefaultSecureKeyValueStore, SecureKeyValueStore} from '../services/MetricsService/SecureKeyValueStorage';
 
 import {Observable} from './Observable';
 import {CheckInData, getNewOutbreakStatus, getOutbreakEvents, initialOutbreakStatus, OutbreakStatus} from './qr';
 import {createCancellableCallbackPromise} from './cancellablePromise';
+import {log} from './logging/config';
+import {getCurrentDate, minutesBetween} from './date-fns';
+
+const OutbreaksLastCheckedDateTime = 'A436ED42-707E-11EB-9439-0242AC130002';
```

Comment:
Let's a Key in the name here so it's more obvious that this is a key vs a data / time